src/seedu/addressbook/commands/SortCommand.java

```diff
@@ -0,0 +1,25 @@
+package seedu.addressbook.commands;
+
+import java.util.List;
+import seedu.addressbook.data.person.ReadOnlyPerson;
+
+
+/**
+ * Sorts all persons in the address book.
+ */
+public class SortCommand extends Command {
+
+    public static final String COMMAND_WORD = "sort";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD
+            + ": Sorts all persons in the address book by alphabetical order.\n"
+            + "Example: " + COMMAND_WORD;
+
+
```

Comment:
This extra empty line can be removed. 😄 
Other than that, could improve by adding unit test for your sort command.